crates/optimism/evm/src/l1.rs

```diff
@@ -68,7 +71,9 @@ pub fn parse_l1_info(input: &[u8]) -> Result<L1BlockInfo, OpBlockExecutionError>
     // If the first 4 bytes of the calldata are the L1BlockInfoEcotone selector, then we parse the
     // calldata as an Ecotone hardfork L1BlockInfo transaction. Otherwise, we parse it as a
     // Bedrock hardfork L1BlockInfo transaction.
-    if input[0..4] == L1_BLOCK_ISTHMUS_SELECTOR {
+    if input[0..4] == L1_BLOCK_INTEROP_SELECTOR {
+        parse_l1_info_tx_ecotone(input[4..].as_ref())
```

Comment:
should this not call a specific interop function or at least the isthmus function?

Reply:
Interop doesn't introduce any changes to parsing I assume, but maybe this should then use the isthmus parse fn?

Reply:
Apologise for the lack of context. Specs seem to be published [here](https://github.com/ethereum-optimism/specs/tree/c67eca6859cb0ddf40b2059a0b9c32fb6c8f8c56/specs/protocol) but I cannot find one an entry for Interop yet.

I am using as a reference the code from op-node [here](
https://github.com/ethereum-optimism/optimism/blob/a33bba2fb32f0ddb350e9b672ac6a7207b607b5e/op-node/rollup/derive/l1_block_info.go#L291-L297). 

You can find an example interop L1 transaction in the Alphanet chain:
```
$ cast tx 0x7d0b4197510307514bf6703183c19d5949490fa25aaa3c79ec9300693ad98a90 --rpc-url https://interop-alpha-0.optimism.io
```

I will add more context to the code and add a unit test with this transaction to validate the decoding.

Reply:
I see that here the interop func is missing operator fields, I don't think this is supposed to be that way, just that it hasn't been updated
https://github.com/ethereum-optimism/optimism/blob/a33bba2fb32f0ddb350e9b672ac6a7207b607b5e/packages/contracts-bedrock/src/libraries/Encoding.sol#L226-L264
at least the interop function will have all params isthmus has
https://github.com/ethereum-optimism/optimism/blob/a33bba2fb32f0ddb350e9b672ac6a7207b607b5e/packages/contracts-bedrock/src/libraries/Encoding.sol#L202-L203

Reply:
for the time being, let's call the isthmus function here not the ecotone